src/common/settings.h

```diff
@@ -172,6 +173,7 @@ struct Values {
 
     // Controls
     InputSetting<std::array<PlayerInput, 10>> players;
+    std::shared_ptr<InputCommon::InputSubsystem> inputSubsystem = NULL;
```

Comment:
```suggestion
    std::shared_ptr<InputCommon::InputSubsystem> input_subsystem;
```

This also shouldn't be inside of the common library.